python/dgl/graph.py

```diff
@@ -3258,7 +3258,7 @@ def filter_edges(self, predicate, edges=ALL):
         """
         if is_all(edges):
             eid = ALL
-            u, v, _ = self._graph.edges()
```

Comment:
Need it to be 'eid' order? By default, the order is arbitrary.

Reply:
I think it has to be eid order, if you check the logic in dgl/udf.py you will find we convert `ALL` placeholder to `F.arange(0, self._g.number_of_edges())`.
On C++ end, by default `edges()` would use `eid` order if order is not specified. But I think we should specify it here if we later changed the default behavior of `edges()`.